microprofile/telemetry/pom.xml

```diff
@@ -140,6 +140,8 @@
                 <configuration>
                     <forkCount>1</forkCount>
                     <reuseForks>false</reuseForks>
+                    <!-- The default for the following is 5000 ms. Reducing it significantly speeds up tests. -->
+                    <argLine>-Dotel.bsp.schedule.delay=100</argLine>
```

Comment:
Does this look sufficient for those days where the pipelines are dog slow?

Reply:
I think this is OK.

This lower setting causes OTel to flush information to our test code much more often than the default. 

The `TestSpanExporter` class internally uses our `MatcherWithRetry` with default timeout of 60 seconds (which we could configure longer if in a pipeline if that became a problem).

The main reason I changed this setting is that the default means flushes occur every 5 seconds, per test, which really prolongs the running time of the tests. This speeds up the running time of those tests quite a bit. I don't think timing out will be a problem, but you're right--with our pipelines, one never knows for sure.